build.sh

```diff
@@ -169,6 +215,33 @@ done
 custom_depenguin_installdir="customfiles/root"
 mkdir -p "$custom_depenguin_installdir"
 
+# setup correct files to copy in based on version
+if [ "$MYVERSION" = "13.2" ]; then
+	if [ -f "$MYCUSTOMDIR/13_2_depenguin_bsdinstall.sh" ]; then
+		cp -f "$MYCUSTOMDIR/13_2_depenguin_bsdinstall.sh" "$MYCUSTOMDIR/depenguin_bsdinstall.sh"
+		chmod +x "$MYCUSTOMDIR/depenguin_bsdinstall.sh"
+	else
+		echo "Missing $MYCUSTOMDIR/13_2_depenguin_bsdinstall.sh"
```

Comment:
errors to stderr ">&2" (won't mention it again in other occurences)

Reply:
will use exit_error function